### DISCUSSION TITLE
Customize Loading Screen

### DISCUSSION THREAD

**Discussion by MSandro** [Sinytra/Connector#293] [Ideas]
Hello, I have tried to change the loading screen.
I know this is nothing relevant at the moment, because there are many other construction sites.

But I have tried Dark Loading Screen (Fabric) and Drippy Loading Screen (Fabric and Forge).
it is possible to load up the pack with the mods, but they don't have any effect.
Drippy Loading Screen Forge works but just after the Connector did its work.

I'm surprised that it loads fabric versions of Fancy Menu and Drippy Loading Screen.
The menu customizations also work as expected, but the loading screen customizations are not respected.

it would be great to have a connector config, to set background color and text / foreground color of the loading screen.

**Comment by Su5eD** [comment]
Hi, the loading screen is handled entirely by Forge, so Connector cannot alter it in any way. We only integrate using the progress bar API that is provided to us.

> it would be great to have a connector config, to set background color and text / foreground color of the loading screen.

For this, I recommend opening a suggestion in the [FancyModLoader](https://github.com/neoforged/FancyModLoader) repository
